custom_components/drivvo/sensor.py

```
@@ -52,80 +50,34 @@ async def async_setup_entry(
-                if description.key == "vehicle":
-                    model = vehicle_data.model
-                    entities.append(
-                        DrivvoSensorEntity(
-                            coordinator,
-                            description,
-                            vehicle_data.id,
-                            vehicle_data.identification,
-                            model,
-                        )
+    entities = []
+    for vehicle_id, coordinator in coordinators.items():
```

Ruff (0.15.12)

[warning] Loop control variable `vehicle_id` not used within loop body

Rename unused `vehicle_id` to `_vehicle_id`

(B007)